gems/pending/spec/spec_helper.rb

```diff
@@ -12,6 +14,8 @@
 # Initialize the global logger that might be expected
 require 'logger'
 $log ||= Logger.new("/dev/null")
+# $log ||= Logger.new(STDOUT)
+# $log.level = Logger::DEBUG
```

Comment:
Since I have a number of changed to make anyway, I can add than now. Is there an environment variable name I should use, or avoid? `SPEC_DEBUG` ? `TEST_DEBUG` ?